### PR TITLE
Support UNIMOText dy2sta

### PR DESCRIPTION
**Pull request by FrostML**
### PR types

New features

### PR changes

Models

### Description

Support UNIMOText dy2sta. 
Delete useless import.

### CODE REVIEW

File: paddlenlp/ops/faster_transformer/sample/unimo_text_export_model_sample.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+from paddlenlp.utils.log import logger
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--model_name_or_path",
+        default="plato-xl",
+        type=str,
+        help="The model name to specify the gpt to use. Can be one of ['gpt2-en', 'gpt2-medium-en', 'gpt-cpm-large-cn']. "
```
Comment (guoshengCS) [inline paddlenlp/ops/faster_transformer/sample/unimo_text_export_model_sample.py]: 这里默认的model_name及帮助信息改成unimo的吧

File: paddlenlp/ops/faster_transformer/sample/unimo_text_inference.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    config.disable_glog_info()
+    predictor = paddle_infer.create_predictor(config)
+
+    input_handles = {}
+    for name in predictor.get_input_names():
+        input_handles[name] = predictor.get_input_handle(name)
+        if name == "attention_mask":
+            input_handles[name].copy_from_cpu(
+                np.asarray(
+                    data[name], dtype="float32").reshape([1, 1, 101, 101]))
```
Comment (guoshengCS) [inline paddlenlp/ops/faster_transformer/sample/unimo_text_inference.py]: 这里的shape还是根据来data来吧

File: paddlenlp/ops/faster_transformer/sample/unimo_text_export_model_sample.py (hunk 1 above)
Comment (FrostML) [reply paddlenlp/ops/faster_transformer/sample/unimo_text_export_model_sample.py]: Fixed. Thanks.

File: paddlenlp/ops/faster_transformer/sample/unimo_text_inference.py (hunk 2 above)
Comment (FrostML) [reply paddlenlp/ops/faster_transformer/sample/unimo_text_inference.py]: Done. Thanks.

**Review by guoshengCS — APPROVED** [review]

**Pull request merged**